lambdas/appointment_email.py

```diff
@@ -145,12 +163,15 @@ def send_cal_email(supp, supp_email, student, student_email, appt_time, duration
         attendees = []
         attendees.append(requester)
     if toCancel:
-        eml_body = appt_type+" appointment between "+supp+" and student "+student+" from "+ddtstart.strftime("%B %d, %Y from %I:%M %p")+" to "+ddtend.strftime("%I:%M %p")+" has been cancelled."
-    
+        eml_body = appt_type+" appointment between "+supp+" and student "+student+" from "+ddtstart.strftime("%B %d, %Y from %I:%M %p")+" to "+ddtend.strftime("%I:%M %p")+" has been cancelled.                                                   "
+        #eml_body = '<html><body><p> appt_type+" appointment between "+supp+" and student "+student+" from "+ddtstart.strftime("%B %d, %Y from %I:%M %p")+" to "+ddtend.strftime("%I:%M %p")+" has been cancelled."</p>' +'<p>"Comment: "+comme </p>' + '</body></html>'
```

Comment:
What is the reason behind adding a line of code and then commenting that line of code in a feature update?

Reply:
_This comment is just an FYI -- no action required!_

This is probably fine for now. But hardcoding email html as a string will get old fast. There are loads of templating languages & packages out there that you can use to store your email templates in a separate file.

Email html / css development is also pretty weird & unique! Email clients actually don't render your email like they would any other web page. They often use wildly outdated rendering engines. Because of this, you have to approach email development in a very unique way from a normal web page. For example, we use [Litmus](https://www.litmus.com/email-testing/) to quickly test an email in each popular email client to see how it renders.

Reply:
Something like [this](https://stackoverflow.com/a/52990429) might be helpful.